config/crd/bases/scheduling.volcano.sh_podgroups.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
why change to use a lower version?

Reply:
in my mechine, run "make update-development-yaml", auto change it to v0.2.4, and use current yaml in master to install volcano， have a error in job api
![image](https://user-images.githubusercontent.com/71266853/115986506-15fd1980-a5e3-11eb-83a2-5df92376f5d1.png)

Reply:
controller-gen version is different between github  and my env， I update my env to v0.4.1